index.html

```diff
@@ -163,63 +162,6 @@ <h3>
         </p>
       </aside>
     </section>
-    <section>
-      <h3>
-        Permissions and user prompts
-      </h3>
-      <p>
-        The [[PERMISSIONS]] API provides a uniform way for websites to request
-        permissions from users and query which permissions they have.
-      </p>
-      <p>
-        A <a>user agent</a> can <dfn data-lt=
-        "deny wake lock|denies the wake lock">deny a wake lock</dfn> of a
-        particular <a>wake lock type</a> for a particular {{Document}} by any
-        implementation-specific reason, such as platform setting or user
-        preference.
-      </p>
-      <p>
-        It is RECOMMENDED that a user agent show some form of unobtrusive
-        notification that informs the user when a wake lock is active, as well
-        as provides the user with the means to [=screen wake lock permission
-        revocation algorithm|block=] the ongoing operation, or simply dismiss
-        the notification.
-      </p>
-      <section>
-        <h2>
-          The `"screen-wake-lock"` powerful feature
```

Comment:
In #324 said  _"If we update the permission text I think we should acknowledge this as an area of future development based on additional implementation experience and that the "screen-wake-lock" permission identifier is a placeholder for that future work"_, I wonder if this is still valid with this patch.

Reply:
Yeah, we should add a note for that.

Reply:
Added a note below about prompting... as it's already a policy controlled feature, we don't need to mention anything  about the "screen-wake-lock" permission.